hs_modflow_modelinstance/models.py

```diff
@@ -338,12 +363,15 @@ def create(cls, **kwargs):
             other_head_dependent_flux_boundary_packages=kwargs.get(
                 'other_head_dependent_flux_boundary_packages', '')
                     )
-        model_boundary_condition._add_specified_head_boundary_packages(
-            kwargs['specified_head_boundary_packages'])
-        model_boundary_condition._add_specified_flux_boundary_packages(
-            kwargs['specified_flux_boundary_packages'])
-        model_boundary_condition._add_head_dependent_flux_boundary_packages(
-            kwargs['head_dependent_flux_boundary_packages'])
+        if kwargs.get('specified_head_boundary_packages'):

```

Comment:
Alternatively you can supply a second, default argument to get i.e. `kwargs.get('key', None)`, like you do below with the `pop`